repo/meta/schema/v3-resource-schema.json

```diff
@@ -9,25 +9,29 @@
       ],
       "properties": {
         "url": {
-          "type": "string"
+          "type": "string",
+          "description": "URL for binary CLI."
         },
         "contentHash": {
           "items": {
             "$ref": "#/definitions/hash"
           },
           "minItems": 1,
-          "type": "array"
+          "type": "array",
+          "description": "List of hashes for binary CLI described in the URL property."
```

Comment:
Suggest: `List of hashes for the binary CLI that are described in the URL property.`